### DISCUSSION TITLE
How to recognize dialog box in Edge with WEBVIEW2 (something like "FindWindow" with webbrowser)?

### DISCUSSION THREAD

**Discussion by Cla222** [MicrosoftEdge/WebView2Feedback#3894] [Q&A]
Hi, I had a project in IE that it used WEBBROWSER component, and FINDWINDOW to recognize the dialog box (the classic "Message from Webpage" to continue or other....)

I had to migrate it for use EDGE in place of IE and I passed to WEBVIEW2 with satisfaction, however (I managed to replicate almost everything)

Unfortunately with FINDWINDOW (and even with the spy++) there is no way to recognize the popup, when - as for IE - it shows up

Does anyone have suggestions for me to overcome the deadlock?

Thanks in advance and excuse me for my bad English 

Claudio

**Comment by champnic** [comment]
@Cla222 Hey Claudio! Are you looking to know when a specific type of dialog box is being shown? Like alerts? You should look at the ScriptDialogOpening event and see if that would work for your scenario:
https://docs.microsoft.com/en-us/nuget/reference/nuget-exe-cli-reference

**Comment by Cla222** [comment]
thank you so much for your reply @champnic

I take a look at the link you sent me asap!
